### ISSUE TITLE
Sound IS possible, is it not.

### ISSUE THREAD

**Issue by wizzwizz4** [LLK/scratch-html5#106]

**Comment by Cythral** [comment]
Sound is only possible upon user interaction on iOS (ie a finger press)

**Comment by wizzwizz4** [comment]
But couldn't you ask for the permission, then keep it until page reload?

**Comment by Cythral** [comment]
Correct, I forgot projects are triggered by touch so yes this is possible

__
Talen Fisher
Cythral

**Comment by wizzwizz4** [comment]
https://github.com/mattdiamond/Recorderjs might help. It abstracts from the default interface, allows for Stereo and Mono, and provides a neat WAV formatter if the player is ever extended to a fully-fledged editor. However, it may need modification if the Stage object is a preferable parent to the Window object.

**Comment by IronManMark20** [comment]
[HowlerJS](https://github.com/goldfire/howler.js/) runs an empty buffer on the first user input. Might be a good work around.

**Comment by wizzwizz4** [comment]
Yes, HowlerJS is better than Recorderjs.

**Comment by IronManMark20** [comment]
Regardless, which ever one we use, we could use that method.

**Comment by Cythral** [comment]
Regardless of what it is, LLK needs to start accepting some of these pull requests before we are able to change anything




Talen Fisher
Cythral Web Services
M: (515) 333-2929
E: vice@cythral.com
Hire me on Freelancer.com

**Comment by wizzwizz4** [comment]
Is a pull request based on a pull request possible?

**Comment by IronManMark20** [comment]
I don't see why not, as long as you fork their fork, not the main repo. It
generally isn't done though.
